assignment/core/src/main/java/com/assignment/core/workflow/PageCreationProcess.java

```diff
@@ -0,0 +1,134 @@
+package com.assignment.core.workflow;
```

Comment:
How will this workflow run on the AEM instance? Add workflow model design and runtime model to code.